packages/medusa/src/api/routes/admin/batch/create-batch-job.ts

```diff
@@ -27,11 +29,20 @@ import { validator } from "../../../../utils/validator"
 export default async (req, res) => {
   const validated = await validator(AdminPostBatchesReq, req.body)
 
+  const batchStrategy: AbstractBatchJobStrategy<never> = req.scope.resolve(
+    `batchType_${validated.type}`
+  )
```

Comment:
we should add some errorhandling to this in order to catch the error that arises when trying to resolve a batchjob that doesn't exist and instead raise an appropriate medusa error.